Tutorial.md

```diff
@@ -0,0 +1,66 @@
+# MSON Tutorial
+
+MSON is short for Markdown Syntax for Object Notation, and it's a way to represent data structures in a human readable plain text form. This tutorial will take you though the basics of MSON and how you can use MSON to describe a data structure.
+
+> **NOTE:** **Additional MSON Resources**
+>
+> + [MSON Specification](https://github.com/apiaryio/mson/blob/master/MSON%20Specification.md)
+> + [Examples](https://github.com/apiaryio/mson#notational-conventions)
+
+## Data structures in MSON
+
+To get started, we're going to create a data structure in MSON to represent a Person.
```

Comment:
Maybe mention here that a person has a name, so it doesn't just show up below. Also, I can't help but post this: http://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/ :tongue: